libnopegl/src/pipeline.h

```diff
@@ -64,8 +64,6 @@ struct pipeline_buffer_desc {
     int binding;
     int access;
     int stage;
-    size_t offset;
-    size_t size;
```

Comment:
Is this also true for the `offset` of the pipeline attribute desc?

Reply:
No.